stglib/exo.py

```diff
@@ -577,210 +578,26 @@ def exo_qaqc(ds):
     [varlist.append(k) for k in ds.data_vars if k not in varlist]
 
     for var in varlist:
-        ds = trim_min(ds, var)
+        ds = qaqc.trim_min(ds, var)
 
-        ds = trim_max(ds, var)
+        ds = qaqc.trim_max(ds, var)
 
-        ds = trim_min_diff(ds, var)
+        ds = qaqc.trim_min_diff(ds, var)
 
-        ds = trim_max_diff(ds, var)
+        ds = qaqc.trim_max_diff(ds, var)
 
-        ds = trim_med_diff(ds, var)
+        ds = qaqc.trim_med_diff(ds, var)
 
-        ds = trim_med_diff_pct(ds, var)
+        ds = qaqc.trim_med_diff_pct(ds, var)
 
-        ds = trim_bad_ens(ds, var)
+        ds = qaqc.trim_bad_ens(ds, var)
 
     for var in varlist:
-        ds = trim_by_any(ds, var)  # re-run and trim by other variables as necessary
+        ds = qaqc.trim_by_any(
+            ds, var
```

Comment:
This seems errant

Reply:
Whoops, disregard. This was a formatting issue.

Reply:
Ok.